src/ray/gcs/tables.cc

```diff
@@ -306,6 +306,13 @@ Status Table<ID, Data>::Subscribe(const JobID &job_id, const ClientID &client_id
       done);
 }
 
+template <typename ID, typename Data>
+Status Table<ID, Data>::Subscribe(const JobID &job_id, const ClientID &client_id,
+                                  const Callback &subscribe,
+                                  const SubscriptionCallback &done) {
+  return Subscribe(job_id, client_id, subscribe, /*failure*/ nullptr, done);
+}
```

Comment:
Doesn't look necessary to add this new `Subscribe` method. Why not just use the original one with nullptr callback at the call sites?

Reply:
It's convenient to reuse  `SubscripitionExecutor`, since `SubscripitionExecutor` used the method `Subscribe(...)` which has four parameters.